plugins/eddn.py

```diff
@@ -331,19 +306,20 @@ def export_outfitting(self, data: Mapping[str, Any], is_beta: bool) -> None:
 
         this.outfitting = (horizons, outfitting)
 
-    def export_shipyard(self, data: Dict[str, Any], is_beta: bool) -> None:
+    def export_shipyard(self, data: CAPIData, is_beta: bool) -> None:
         """
         export_shipyard updates EDDN with the current (lastStarport) station's outfitting options, if any.
         once the send is complete, this.shipyard is updated to the new data.
 
         :param data: dict containing the shipyard data
         :param is_beta: whether or not we are in beta mode
         """
-        modules, ships = self.safe_modules_and_ships(data)
+
+        ships = data['lastStarport']['ships']
 
         horizons: bool = is_horizons(
             data['lastStarport'].get('economies', {}),
-            modules,
+            data['lastStarport']['modules'],
             ships
```

Comment:
If we don't need the temporary var for modules then we don't need it now for ships either.

Reply:
Is used below in the construction of the Shipyard list (L326-331)